index.js

```diff
@@ -194,8 +220,15 @@ export function react(text, opts = {}, components = {}) {
         img: Image(sanitize),
         ...components,
       }),
-    })
-    .processSync(text).contents;
+    });
+
+  const tree = reactProcessor.parse(text);
+  const toc = reactTOC(tree, opts);
+
+  return {
```

Comment:
I think we run a bit of a risk changing the return value like this, since it'd be a breaking change on what is, essentially, the default export method! I've already started refactoring this in to it's own method, `reactTOC()`, above. Will update [the integration branch](/readmeio/readme/pull/2965) to reflect as well.

Reply:
Done in f9bcec2.